pkg/clients/clients.go

```diff
@@ -1,14 +1,17 @@
 package clients
 
 import (
-	"github.com/coreos/etcd/client"
-	"k8s.io/client-go/rest"
-	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	etcd "github.com/coreos/etcd/client"
+	k8s "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	"sync"
```

Comment:
I literally can't save the file and format it this way with my current setup. vim-go orders alphabetically. I'll follow up with you to get my env using goimports, it's always been broken for me.